modules/upload-kernel-support-package-to-central.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * configuration/enable-offline-mode.adoc
+:_module-type: PROCEDURE
+[id="upload-kernel-support-package-to-central_{context}"]
+= Uploading kernel support packages to Central
+
+You can upload the kernel support packages containing probes for all recent and supported Linux kernel versions to Central.
+
+.Prerequisites
+
+* You must have an API token with the administrator role.
+* You must have installed the `roxctl` command-line interface (CLI).
+
+.Procedure
+
+. Set the `ROX_API_TOKEN` and the `CENTRAL_ADDRESS` tokens:
++
+[source,terminal]
+----
+$ export ROX_API_TOKEN=<api_token>
+----
++
+[source,terminal]
+----
+$ export CENTRAL_ADDRESS=<address>:<port_number>
+----
+. Run the following command to upload the kernel support packages:
++
+[source,terminal]
+----
+$ roxctl collector support-packages upload <package_file> \
```

Comment:
This command doesn't use the two variables defined above. Just checking if that's right or not.

Reply:
Thanks yes, but roxctl uses the other variable `ROX_API_TOKEN` internally to connect to Central and do operations.